### PR TITLE
wip: Added docs website landing page

### PR DESCRIPTION
**Pull request by way2nizam**
nyaa-ui component website landing page.

1. index.html
2. landingpage.css

Screenshot from 2022-02-05 17-26-43

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,24 +7,81 @@
...
+            <div class="footer__brand">
+                <p class="footer__copyright">Made with Love ♥ | Feel free to copy.</p>
+            </div>
+            <div class="footer__social">
+                <a href="https://github.com/way2nizam" class="footer__icon"><img src="./images/github.svg" alt="">
+                </a>
+                <a href="https://twitter.com/way2nizam" class="footer__icon"><img src="./images/twitter.svg" alt="">
+                </a>
+                <a href="https://www.linkedin.com/in/nizamy710/" class="footer__icon"><img src="./images/linkedin.svg"
+                        alt="">
```
Comment (devananth) [inline index.html]: Add `alt` attribute value in the `img` tags

**Review by devananth — COMMENTED** [review]
Great work ! UI is clean

File: index.html (hunk 1 above)
Comment (way2nizam) [reply index.html]: thanks devananth, I'll add.

**Review by tanishk15sharma — APPROVED** [review]
good to merge !

**Pull request merged**